app/code/Magento/InventoryInStorePickupFrontend/Block/Checkout/LayoutProcessor.php

```diff
@@ -0,0 +1,204 @@
+<?php
```

Comment:
Please, revert checkout layout xml and add only missing data in the LayoutProcessor

Reply:
Done.